projects/plugins/boost/app/assets/src/js/react-components/BoostPricingTable.tsx

```diff
@@ -52,6 +52,18 @@ export const BoostPricingTable = ( { pricing, onPremiumCTA, onFreeCTA } ) => {
 		onFreeCTA();
 	};
 
+	const discountPercentage =
+		pricing.yearly.priceBefore !== undefined && pricing.yearly.priceAfter !== undefined
+			? Math.floor(
+					( ( pricing.yearly.priceBefore - pricing.yearly.priceAfter ) /
+						pricing.yearly.priceBefore ) *
+						100
+			  )
+			: 0;
+
+	// If the first year discount ends, we want to remove the label without updating the plugin.
+	const promoLabel = discountPercentage === 50 ? __( 'First Year Discount', 'jetpack-boost' ) : '';
```

Comment:
$AUD doesn't discount exactly 50%. I don't think we can use the exact discount amount as a gate here.

However, if you look at `Wpcom_Products::get_product_pricing`, `discount_price` only differs from `full_price` if there is introductory pricing. It shows how to read the `introductory_pricing`.

IMHO we can't guarantee that `Wpcom_Products::get_product_pricing` will only ever return `introductory` pricing - we may have other reasons for discount in the future.

So I think the "correct" fix is to add a method to `Wpcom_Products` which returns true if there is introductory pricing, so that the UI can show that badge. Look at `Wpcom_Products::get_product_pricing` for how to check for the presence of intro pricing.

It means you'll need a review from someone with approver rights over My-Jetpack, but I'm pretty sure @oskosk can help with that ;)